### ISSUE TITLE
biojava 3.0.6 no longer available?

### ISSUE THREAD

**Issue by alexagrf** [ssadedin/groovy-ngs-utils#1]
./gradlew jar fails trying to find biojava 3.0.6
```

FAILURE: Build failed with an exception.

* Where:
Build file '/home/dubrova/src/groovy-ngs-utils/build.gradle' line: 63

* What went wrong:
A problem occurred evaluating root project 'groovy-ngs-utils'.
> Could not resolve all dependencies for configuration ':compile'.
   > Could not find org.biojava:biojava3-core:3.0.6.
     Searched in the following locations:
         http://repo1.maven.org/maven2/org/biojava/biojava3-core/3.0.6/biojava3-core-3.0.6.pom
         http://repo1.maven.org/maven2/org/biojava/biojava3-core/3.0.6/biojava3-core-3.0.6.jar
         http://www.biojava.org/download/maven/org/biojava/biojava3-core/3.0.6/biojava3-core-3.0.6.pom
         http://www.biojava.org/download/maven/org/biojava/biojava3-core/3.0.6/biojava3-core-3.0.6.jar
         http://mvn.erichseifert.de/maven2/org/biojava/biojava3-core/3.0.6/biojava3-core-3.0.6.pom
         http://mvn.erichseifert.de/maven2/org/biojava/biojava3-core/3.0.6/biojava3-core-3.0.6.jar
     Required by:
         :groovy-ngs-utils:unspecified
   > Could not find org.biojava:biojava3-alignment:3.0.6.
     Searched in the following locations:
         http://repo1.maven.org/maven2/org/biojava/biojava3-alignment/3.0.6/biojava3-alignment-3.0.6.pom
         http://repo1.maven.org/maven2/org/biojava/biojava3-alignment/3.0.6/biojava3-alignment-3.0.6.jar
         http://www.biojava.org/download/maven/org/biojava/biojava3-alignment/3.0.6/biojava3-alignment-3.0.6.pom
         http://www.biojava.org/download/maven/org/biojava/biojava3-alignment/3.0.6/biojava3-alignment-3.0.6.jar
         http://mvn.erichseifert.de/maven2/org/biojava/biojava3-alignment/3.0.6/biojava3-alignment-3.0.6.pom
         http://mvn.erichseifert.de/maven2/org/biojava/biojava3-alignment/3.0.6/biojava3-alignment-3.0.6.jar
     Required by:
         :groovy-ngs-utils:unspecified

```

Switching to 3.0.8 (the version that seemed closest on the biojava site) on build.gradle doesn't help, similar message is shown.

**Comment by ssadedin** [comment]
It seems like the BioJava site is doing some major renovations and breaking all their download links seems to have been a consequence. Even the direct download links on the Biojava site were broken the other day. I see they have removed them altogether now. For now I am just waiting this out ... because I'm hoping they will put back links to 3.0. If they don't maybe I will try out 4.0 but I 

If you can get hold of the jar file, just delete the biojava maven references and put it directly in the "lib" directory and I think things should compile.

**Comment by ssadedin** [comment]
I have just pushed a new branch that upgrades both biojava and picard tools and should build cleanly from artifacts in maven. It's worth noting that in both cases they moved packages around which means it results in some package changes for classes that are part of the APIs in gngs-utils (ie: this can break backwards compatibility). I limited the picard upgrade to 1.141 because that is the last version supporting JDK7.

For now I'm closing this as I don't plan to go back and address the problem for previous versions, however for people wishing to build the old versions the strategy of placing the biojava3 artefacts into the lib directory and removing the entries from build.gradle should work.
